demo-project/src/main/resources/application.properties

```diff
@@ -11,3 +11,5 @@ spring.flyway.locations=classpath:flyway/migrations
 
 spring.rabbitmq.host=localhost
 spring.rabbitmq.port=5672
+spring.rabbitmq.publisher-confirm-type=correlated
```

Comment:
According to [the messy documentation](https://docs.spring.io/spring-amqp/docs/current/reference/html/#scoped-operations), this can just be `simple`. It's not entirely clear what "infrastructure is avoided", and the [source code](https://github.com/spring-projects/spring-amqp/blob/f52a671ff4d490433f1790f9bc8343f66cada650/spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/CachingConnectionFactory.java#L145) isn't terribly helpful either.

But, `simple` does work here (I've just tested it), so let's go with that.